package.json

```diff
@@ -45,6 +45,7 @@
     "eventemitter3": "^5.0.1",
     "jszip": "^3.10.1",
     "peerjs": "^1.5.1",
+    "qrcode": "^1.5.3",
```

Comment:
Our goal is to allow non-PC devices (such as phones) to get bootstrapped using QR code. Therefore, you need to use a SCANNER library, rather than a generator.